docs/explanations/motd_messages.md

```diff
@@ -104,8 +135,59 @@ coverage for your applications.
 Your grace period will expire in 9 days.
 ```
 
-### How are these messages updated and inserted into MOTD?
+## MOTD for the ESM announcement
+
+There was also a general announcement info that further helped to raise
+awareness of Pro being available and for free. This was only shown on systems
+covered by `esm-apps`. Furthermore it was meant to be only shown for a limited
+time and therefore removed in 27.14.
+It looked like:
+
+```
+ * Introducing Expanded Security Maintenance for Applications.
+   Receive updates to over 25,000 software packages with your
+   Ubuntu Pro subscription. Free for personal use.
+
+     https://ubuntu.com/pro
+```
+
```

Comment:
Since this message was intended as a limited-time announcement to coincide
with the release of Ubuntu Pro into general availability, it was removed in
27.14.

Reply:
done